src/custom.scss

```diff
@@ -3,12 +3,42 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
```

Comment:
is this needed? what's this for?

Reply:
idk I just copied everything from index.css into here

Reply:
getting rid of it broke nothing so 🤷‍♂️